Spanner/src/Database.php

```diff
@@ -183,14 +183,16 @@ public function __construct(
         $projectId,
         $name,
         SessionPoolInterface $sessionPool = null,
-        $returnInt64AsObject = false
+        $returnInt64AsObject = false,
+        array $info = null
```

Comment:
You could default the array in the signature here, instead of needing the elvis operator on line 195

```suggestion
        array $info = []
```

Reply:
Done, thanks